src/Api.Rest/Contracts/ICustomRestClient.cs

```diff
@@ -0,0 +1,55 @@
+namespace Zeiss.PiWeb.Api.Rest.Common.Contracts;

```

Comment:
I think this namespace is not correct anymore, since the file is in the contracts folder outside of common.